jirazzz

```diff
@@ -412,6 +418,35 @@
                  xs))
           ""))))
 
+(defn mine
+  {:command 'mine
+   :doc "List open issues assigned to me"}
+  [opts]
+  (let [{:keys [total issues]}
+        (jira-req opts
+          :get (str "/rest/api/2/search?jql=" (java.net.URLEncoder/encode "assignee=currentuser() AND status != Closed")))]
+    (->> issues
+         (map (fn [{key :key, {:keys [status summary]} :fields}]
+                {:key key
+                 :status (:name status)
+                 :summary (subs summary 0 (min (count summary) 40)) 
+                 :browse (jira-url opts "/browse/ARD-10019")}))
```

Comment:
should this path be something like `(str "/browse/" key)` ?

Reply:
no, you need the whole url so that you can click it in the terminal to open it in a browser

Reply:
yes, the whole url is needed, but surely the path shouldn't be hard-coded?

Reply:
🤦
Fixed